bluetooth: Bluetooth GATT server serialization (#4984)
nrfconnect/sdk-nrf: samples/bluetooth/rpc_host/README.rst

See :ref:`configure_application` for information about how to configure a sample.

Build the sample with the same Bluetooth configuration as the application core sample.
For more details, see: :ref:`ble_rpc`.
Build the :ref:`peripheral_uart` on the application core.
This sample works out of the box and does not require configuration changes.
In the Peripheral UART sample directory, invoke:

.. code-block:: console

   west build -b nrf5340dk_nrf5340_cpuapp -- -DCONFIG_BT_RPC=y

> You can also build :ref:`peripheral_hids_mouse` using the above command.
> This sample works also out of the box but it contains additional :file:`samples/bluetooth/peripheral_hids_mouse/child_image/rpc_host.conf` which is used to configure this sample.
> You can take it as an example on how to create a configuration for the your own application.

Reviewer: ```suggestion
You can also build :ref:`peripheral_hids_mouse` using the above command.
This sample requires some additional configuration in the :file:`samples/bluetooth/peripheral_hids_mouse/child_image/rpc_host.conf` file.
You can take it as an example on how to create configuration for your own application.
```
